enigma-swing/src/main/java/cuchaz/enigma/gui/GuiController.java

```diff
@@ -301,7 +301,7 @@ public void openReference(EntryReference<Entry<?>, Entry<?>> reference) {
 	 * @param reference the reference
 	 */
 	private void setReference(EntryReference<Entry<?>, Entry<?>> reference) {
-		gui.openClass(reference.getLocationClassEntry()).showReference(reference);
+		gui.openClass(reference.getLocationClassEntry().getOutermostClass()).showReference(reference);
```

Comment:
I think this is actually the fix for it opening a subclass in a new tab instead of the outer class's tab that I noticed in #331! Nice